scripts/untar_gz_files.py

```diff
@@ -52,27 +53,28 @@ def main(bucket: str, subdir: str, blob_name: str, outdir: str):
     )
     logging.info(f'Untared {blob_name}')
 
-    extracted_from_tarball = os.listdir(f'./{subdir}/extracted')
+    # Recursively get all paths to everything extracted from tarball
+    extracted_from_tarball = pathlib.Path(f'./{subdir}/extracted')
+    all_extracted_paths = [
+        str(path) for path in list(extracted_from_tarball.rglob('*'))
+    ]
+
+    # Discard paths that are directories, keep paths to files
+    extracted_file_paths = []
+    for extracted_path in all_extracted_paths:
+        if not os.path.isdir(extracted_path):
+            extracted_file_paths.append(extracted_path)
```

Comment:
Chuck all this in a conditional list comprehension

```suggestion
    extracted_file_paths = [
        str(path) for path in extracted_from_tarball.rglob('*') 
        if not path.is_dir()
    ]
```